public/js/components/AtomEdit/CustomEditors/GuideEditor.js

```diff
@@ -1,30 +1,36 @@
-import React, { PropTypes } from 'react';
-import FormFieldImageSelect from '../../FormFields/FormFieldImageSelect';
-import FormFieldArrayWrapper from '../../FormFields/FormFieldArrayWrapper';
-import {GuideItem} from './GuideFields/GuideItem';
-import {ManagedField, ManagedForm} from '../../ManagedEditor';
-import {atomPropType} from '../../../constants/atomPropType';
-import {checkItemsUnderWordCount} from '../../../util/validators';
+import React, { PropTypes } from "react";
+import FormFieldImageSelect from "../../FormFields/FormFieldImageSelect";
+import FormFieldArrayWrapper from "../../FormFields/FormFieldArrayWrapper";
+import { GuideItem } from "./GuideFields/GuideItem";
+import { ManagedField, ManagedForm } from "../../ManagedEditor";
+import { atomPropType } from "../../../constants/atomPropType";
 
 export class GuideEditor extends React.Component {
-
   static propTypes = {
     atom: atomPropType.isRequired,
     onUpdate: PropTypes.func.isRequired,
     onFormErrorsUpdate: PropTypes.func,
     config: PropTypes.shape({
-      gridUrl: PropTypes.string.isRequired
-    }).isRequired
-  }
+      gridUrl: PropTypes.string.isRequired,
+    }).isRequired,
+  };
 
-  render () {
+  render() {
     return (
       <div className="form">
-        <ManagedForm data={this.props.atom} updateData={this.props.onUpdate} onFormErrorsUpdate={this.props.onFormErrorsUpdate} formName="guideEditor">
-          <ManagedField fieldLocation="data.guide.guideImage" name="Guide Image">
-            <FormFieldImageSelect gridUrl={this.props.config.gridUrl}/>
+        <ManagedForm
+          data={this.props.atom}
+          updateData={this.props.onUpdate}
+          onFormErrorsUpdate={this.props.onFormErrorsUpdate}
+          formName="guideEditor"
+        >
+          <ManagedField
+            fieldLocation="data.guide.guideImage"
+            name="Guide Image"
+          >
+            <FormFieldImageSelect gridUrl={this.props.config.gridUrl} />
           </ManagedField>
-          <ManagedField fieldLocation="data.guide.items" name="Items" customValidation={[checkItemsUnderWordCount]}>
+          <ManagedField fieldLocation="data.guide.items" name="Items">
```

Comment:
Here. Is that correct?

Reply:
yes! instead of attaching a validator to check for the word limit to error the form we are using a prop in the textfield (This was already built before I joined, super handy!) to show a notice if a word limit is hit. The reason for this is that word limits aren't errors but more of a suggestion!